dbt/include/spark/macros/adapters.sql

```diff
@@ -179,12 +179,18 @@
   {{ return(adapter.get_columns_in_relation(relation)) }}
 {% endmacro %}
 
-{% macro spark__list_relations_without_caching(relation) %}
-  {% call statement('list_relations_without_caching', fetch_result=True) -%}
-    show table extended in {{ relation }} like '*'
+{% macro spark__list_tables_without_caching(relation) %}
```

Comment:
Could you use dispatch pattern?

```
{% macro list_tables_without_caching(relation) %}
  {{ return(adapter.dispatch('list_tables_without_caching', 'dbt')(relation)) }}
{%- endmacro -%}

{% macro spark__list_tables_without_caching(relation) %}
...
```

Reply:
Sure, thanks for your comment, I will update it soon.